ci/cudf_pandas_scripts/run_tests.sh

```diff
@@ -91,7 +91,7 @@ IFS=',' read -r -a versions <<< "$output"
 
 for version in "${versions[@]}"; do
     echo "Installing pandas version: ${version}"
-    python -m pip install "numpy>=1.23,<2.0a0" "pandas==${version}"
+    python -m pip install "numpy>=1.23,<2.0a0" "pandas==${version}.*"
```

Comment:
`version` here is always of the form `{major}.{minor}`, populated by this script that reads `dependencies.yaml`:

https://github.com/rapidsai/cudf/blob/7018a33be752da9363db5431560d8d12bf378920/ci/cudf_pandas_scripts/fetch_pandas_versions.py#L15